v2/src/layer.ts

```diff
@@ -41,7 +41,8 @@ export function applyLayers(
       lam.architecture?.dockerPlatform !== undefined &&
       lam.architecture.dockerPlatform === Architecture.ARM_64.dockerPlatform;
     const isNode = lambdaRuntimeType === RuntimeType.NODE;
-    if (lambdaRuntimeType === RuntimeType.UNSUPPORTED) {
+
+    if (lambdaRuntimeType === undefined || lambdaRuntimeType === RuntimeType.UNSUPPORTED) {
       log.debug(`Unsupported runtime: ${runtime}`);
```

Comment:
Just checking but what's the value of `runtime` if `lambdaRuntimeType` is undefined? Is that just going to print undefined?

Reply:
Yeah for container images it's also undefined

Reply:
LGTM